devices/Sen5x/Extensions/SpanByteExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+using System;
+
+namespace Iot.Device.Sen5x.Extensions
+{
+    internal static class SpanByteExtensions
+    {
+        /// <summary>
+        /// A convenient method that verifies all pairs within a data span. The data must be a multiple of 3 bytes (2 bytes data, 1 byte crc).
+        /// </summary>
+        /// <param name="data">The data to be verified.</param>
+        /// <exception cref="ArgumentException">When the byte lenght is not a multiple of 3.</exception>
+        /// <exception cref="IndexOutOfRangeException">When the CRC fails.</exception>
+        internal static void VerifyCrc(this SpanByte data)
+        {
+            if (data.Length % 3 != 0)
+            {
+                throw new ArgumentException(nameof(data), "Bytes in data must be a multiple of 3");
```

Comment:
you can remove the text of the exception to gain space in the generated PE file. You've already added them in the intellisense comment. We're levering this pattern. So except if the comment is a real must, please remove them from the exceptions in all the project. But make sure you have them in the intellisense comments